app/pages/user-reservations/reservation-list/ReservationListContainer.js

```diff
@@ -31,39 +30,32 @@ class UnconnectedReservationListContainer extends Component {
   render() {
     const {
       emptyMessage,
-      loading,
       reservations,
       page,
       pages,
       t,
     } = this.props;
 
+    if (reservations.length <= 0) return <p>{emptyMessage || t('ReservationListContainer.emptyMessage')}</p>;
+
     return (
-      <Loader loaded={!loading}>
-        {reservations.length > 0
-          ? (
-            <div>
-              <ul className="reservation-list">
-                {reservations.map(this.renderReservationListItem)}
-              </ul>
-              <Pagination
-                onChange={this.props.onPageChange}
-                page={page}
-                pages={pages}
-              />
-            </div>
-          )
-          : <p>{emptyMessage || t('ReservationListContainer.emptyMessage')}</p>
-        }
-      </Loader>
+      <>
```

Comment:
Should this be a div (as before the change) or is a fragment intended here due to new styling?